src/styles/index.css

```diff
@@ -0,0 +1,239 @@
+html {
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px #eee;
+}
+
+.header__logo {
+  height: 40px;
+  width: 40px;
+}
+
+.header__navigation {
+  margin: 0;
+}
+
+.main {
+  margin: 0;
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: 200px;
+  justify-content: center;
+  gap: 46px 48px;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.navigation__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style-type: none;
+  align-items: end;
+}
+
+.navigation__item {
+  text-transform: uppercase;
+}
+
+.navigation__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.navigation__link {
+  display: block;
+  height: 60px;
+  line-height: 60px;
+  text-decoration: none;
+  color: #060b35;
+}
+
+.navigation__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: '';
+  width: 37px;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: scale 0.3s;
+}
+
+.card:hover {
+  scale: 1.2;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto;
+  width: 160px;
+  height: 134px;
+}
+
+.card__name {
+  margin: 40px 0 0;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  letter-spacing: 0%;
+  color: #060b35;
+}
+
+.card__code {
+  margin: 4px 0 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  letter-spacing: 0%;
+  color: #616070;
+}
+
+.card__rates {
+  margin: 16px 0 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__rates-reviews {
+  margin: 0;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 16px;
+  letter-spacing: 0%;
+  color: #060b35;
+}
+
+.card__price {
+  margin: 24px 0 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-text {
+  margin: 0;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  letter-spacing: 0%;
+  color: #616070;
+}
+
+.card__price-cost {
+  margin: 0;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  letter-spacing: 0%;
+  color: #060b35;
+}
+
+.card__button {
+  height: 40px;
+  margin-top: 16px;
+  padding: auto 0;
```

Comment:
The 'padding' property value 'auto 0' is incorrect. 'auto' is not a valid value for padding. Consider using a specific unit like 'px', 'em', or '%' for both top and bottom padding.